src/sites/link/1ink.cc.js

```diff
@@ -0,0 +1,13 @@
+_.register({
+  rule: {
+    host: /^1ink\.cc$/,
+    path: /^\/\w+$/,
+  },
+  async ready () {
+    let url = document.head.querySelector('[name=keywords]').content;
+    const urlCheck = url.includes('http');
```

Comment:
What if the URL actually contains `http` string? (e.g. `example.com/http.html`)
We probably want something like: `url.indexOf('http') === 0` instead.

Reply:
Already changed the string to 'http://' and add one more comparison for https.
So it will safe.